staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -952,12 +976,20 @@ func ValidateCustomResourceDefinitionOpenAPISchema(schema *apiextensions.JSONSch
 		}
 
 		structural, err := structuralschema.NewStructural(schema)
+		celCRDCost := uint64(0)
```

Comment:
`expressionCost` ?

Reply:
Initializing `celCRDCost` means that we are summing the costs of all the rules of a x-kubernetes-validations extensions and checking if the total of their combined costs exceeds a per-expression limit.

Instead, I was expecting that we would check just the result of `getExpressionCost` against the pre-expression limit.

Reply:
Right - I confused the per-request part of the KEP here. Fixed